src/Database/Adapter/PdoAdapter.php

```diff
@@ -403,6 +408,50 @@ private function isMulti(array $data): bool
         return true;
     }
 
+    private function splitColumnNameAndOperator(string $key, $value, string &$columnName, string &$operator): void
```

Comment:
please add phpdoc typehint mixed for $value

Reply:
$value has no typehint in many places, before my edit. Is it needed everywhere?

Reply:
There should be typehints everywhere - at least php doc (for mixed types) because phpstan requires them. I have to check it but I am pretty sure that it is not missing anywhere just here in your new method

Reply:
I saw it missing somewhere, I'll add them